### PR TITLE
WD-8394 - Add secrets tab

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Add a component for the secrets tab.
- Linked to the secrets tab from the model details.

## QA

- Go to the details page for a model.
- Check that there is a "Secrets tab".
- Clicking on the secrets tab should take you to a blank page.

## Details

https://warthogs.atlassian.net/browse/WD-8394

### CODE REVIEW

File: src/pages/EntityDetails/Model/Model.tsx [hunk 1]
```diff
@@ -242,6 +243,7 @@ const Model = () => {
           </>
         )}
         {shouldShow("logs", query.activeView) && <Logs />}
+        {shouldShow("secrets", query.activeView) && <Secrets />}
```
Comment (vladimir-cucu) [inline src/pages/EntityDetails/Model/Model.tsx:246]: I think we might need to add `case "secrets":` in the definition of `shouldShow`. Currently, the `Secrets` page doesn't render.

File: src/pages/EntityDetails/EntityDetails.tsx [hunk 2]
```diff
@@ -138,6 +138,13 @@ const EntityDetails = ({ modelWatcherError }: Props) => {
         to: urls.model.tab({ userName, modelName, tab: ModelTab.LOGS }),
         component: Link,
       },
+      {
+        active: activeView === "secrets",
+        label: "Secrets",
+        onClick: (e: MouseEvent) => handleNavClick(e),
```
Comment (vladimir-cucu) [inline src/pages/EntityDetails/EntityDetails.tsx:144]: Should we add a test here?

**Review by vladimir-cucu — APPROVED** [review]
Looks good to me! :rocket:

File: src/pages/EntityDetails/Model/Model.tsx (hunk 1 above)
Comment (huwshimi) [reply src/pages/EntityDetails/Model/Model.tsx:246]: Oops, thanks!

File: src/pages/EntityDetails/EntityDetails.tsx (hunk 2 above)
Comment (huwshimi) [reply src/pages/EntityDetails/EntityDetails.tsx:144]: Done.

**Pull request merged**